src/components/10-atoms/checkbox/react/story.jsx

```diff
@@ -20,6 +20,7 @@ storiesOf('Atoms/Checkbox/React', module)
     const disabled = boolean('disabled', false);
     const errortext = boolean('error', false);
     const required = boolean('required', false);
+    const variant = text('varinant', 'square');
```

Comment:
Use knob type `dropdown` if you just allow a set of values. 

Typo in `varinant`

Reply:
Thanks 👍